.github/workflows/ci.yml

```diff
@@ -20,5 +20,61 @@ jobs:
           - beta
           - nightly
     steps:
-      - uses: actions/checkout@v4
-      - run: bash scripts/ci.sh
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Tests
+        run: |
+          cd catlog
+
+          rustup update ${{ matrix.toolchain }} && rustup default ${{ matrix.toolchain }}
+          cargo build --verbose
+          cargo test --verbose
+
+  catlog_formatting:
+    name: Catlog Formatting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Check formatting
```

Comment:
We should also run `cargo fmt --check` on the auxiliary crate in `frontend/catlog-wasm`.

Reply:
This is now in #42!